include/soci/session.h

```diff
@@ -232,6 +236,8 @@ class SOCI_DECL session
     bool isFromPool_;
     std::size_t poolPosition_;
     connection_pool * pool_;
+
+    struct schema_table_name * schema_table_name_;
```

Comment:
We should initialize members in their declaration:

```suggestion
    schema_table_name * schema_table_name_ = nullptr;
```